# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
bool 	success					# true/false
uint32 	cloud_id				# id of the point cloud (counter since last reset)
time 	scan_request_received	# Timestamp for when the scan request was recieved by the 3D sensor
float32   scan_time				# Scan time in milliseconds
sensor_msgs/PointCloud2 pc		# Point Cloud from the Scan process..
